modes/magit/evil-collection-magit.el

```diff
@@ -319,6 +319,7 @@ moment.")
        (,states magit-status-mode-map "gh"  magit-section-up                       "^")
        (,states magit-diff-mode-map "gj" magit-section-forward)
        (,states magit-diff-mode-map "gd" magit-jump-to-diffstat-or-diff "j")
+       ((visual) magit-diff-mode-map "y" magit-copy-section-value)
```

Comment:
Do you have a simple set of instructions to get to magit-diff-mode?

Reply:
> Do you have a simple set of instructions to get to magit-diff-mode?

In a magit-status buffer, navigate to a commit line, then invoke `dd`. This will open a magit-diff buffer.